src/movement.cpp

```diff
@@ -820,16 +820,16 @@ bool MoveEvent::executeStep(Creature* creature, Item* item, const Position& pos)
 uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool boolean)
 {
 	if (scripted) {
-		return executeEquip(player, item, slot);
+		return executeEquip(player, item, slot, boolean);
```

Comment:
While you're at it, you could change the name of the boolean variable to something that makes more sense.